Automattic/jetpack
# [Boost] Update copy around Cloud/Critical CSS

Fixes https://github.com/Automattic/boost-cloud/issues/216

## Proposed changes:


Project: pbNhbs-5b8-p2
Design: pbNhbs-5lC-p2

Update Boost's copy around critical/cloud CSS generation. More specifically:

- Update Getting Started page wording related to Cloud/Critical CSS generation;
- Update Getting Started page to include tooltips for Cloud/Critical CSS generation;
- Add green checkmark in front of "Must be done manually";
- Update wording on Boost's settings page related to Cloud/Critical CSS generation;
- Add additional paragraph below Critical/Cloud CSS feature description (with a tooltip, for the free version, explaining why regeneration is necessary);
- Update status text (8 files generated 7 days ago.) to include a regeneration prompt note;
- Move upgrade CTA on Boost's settings page to right below the Critical CSS option.

Notes:
- Pricing widget should remain the same (pbNhbs-5lC-p2#comment-13730);
- Tooltip close buttons present in the designs have been removed to keep the same design across the board (pbNhbs-5lC-p2#comment-13729).
- Green checkmarks on getting started page are slightly different than the one in the design. Those have been left as is because of the note above.
- Title spacing below new getting started tooltips has not been changed;
- Bottom link coloring on getting started page has not been changed.

Other:
- Due to the new tooltips on the getting started page being wider than the rest, the Pricing Table component has been updated accordingly. Should be reviewed by crew after team.

### Other information:


## Jetpack product discussion



Project: pbNhbs-5b8-p2
Design: pbNhbs-5lC-p2

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup this version on your website;
* Make sure getting started page has the necessary changes;
* Make sure the main Boost settings page has the necessary changes;
* Make sure to test both free and premium versions (best done with [Boost companion](https://github.com/Automattic/boost-developer) plugin).
